src/components/Article/Article.jsx

```diff
@@ -4,3 +4,72 @@
 // Create an Article function returning the HTML of article block
 
 // Add a default export statement for Article component to use it in the other files
+
+import React from 'react';
+import './Article.css';
+
+const Article = () => (
+  <React.Fragment>
```

Comment:
```suggestion
  <>
```
You can use empty braces instead of using react.fragment:)

Reply:
but in this case, you don't need React.Fragment at all!
you already have one parent element and it's `<article></article>`.